policyengine_us/parameters/gov/states/ca/cdss/capi/payment_standard_offset/couple.yaml

```
@@ -0,0 +1,10 @@
+description: California reduces the SSI/SSP payment standard by this amount for couples under the CAPI program.
```

Generally avoid acronyms in descriptions (keep in labels)
```suggestion
description: California reduces the Supplemental Security Income / State Supplemental Payment payment standard by this amount for couples when calculating the Cash Assistance Program for Immigrants.
```